compositor_render/src/transformations/shader_executor.rs

```diff
@@ -0,0 +1,182 @@
+use crate::renderer::{
```

Comment:
why? what is shader_executor? What is suppose to be in this module?

Reply:
Renamed to gpu_shader